src/hacbs/components/Commits/CommitsGettingStartedModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Button, Modal, ModalVariant, Text, Title } from '@patternfly/react-core';
+import imageUrl from '../../imgs/app-workflow.svg';
+
+type CommitsGettingStartedModalProps = {
+  shown: boolean;
+  onHide: () => void;
+};
+
+const CommitsGettingStartedModal: React.FC<CommitsGettingStartedModalProps> = ({
+  shown,
+  onHide,
+}) => {
+  const header = (
+    <Title className="pf-u-text-align-center" headingLevel="h2">
+      Getting started with Commit view
+    </Title>
+  );
+
+  const body = (
+    <>
+      <div className="pf-u-display-flex pf-u-justify-content-center pf-u-flex-fill">
+        <img src={imageUrl} alt="getting started with commits" />
+      </div>
+      <Text className="pf-u-text-align-left pf-u-mt-md">
+        View Commit activity, triggered pipelines and environment deployment in a single view.
+        <br />
+        Each node represents a github action, pipeline or an environment.
+      </Text>
```

Comment:
Not that we have this interaction right now, but it's in the design.
```suggestion
        <br />
        Click on nodes to view more details.
      </Text>
```